### PR TITLE
Add ts_package extension

### PR DESCRIPTION
**Pull request by higebu**
This PR adds the `grpc.gateway.protoc_gen_grpc_gateway_ts.options.ts_package` option to specify the package name of proto files.

For example, we can add following lines to `google/protobuf/empty.proto` with package name "google-protobuf/google/protobuf/empty_pb".
Then we can use [@types/google-protobuf](https://www.npmjs.com/package/@types/google-protobuf) package for `google.protobuf.Empty`.

```protobuf
import "protoc-gen-grpc-gateway-ts/options/ts_package.proto";
option (grpc.gateway.protoc_gen_grpc_gateway_ts.options.ts_package) = "package_name";
```

Related to: https://github.com/protocolbuffers/protobuf/issues/6341

### CODE REVIEW

File: registry/registry.go
Comment (lyonlai) [inline registry/registry.go:347]: Could we have a better debug message? saying `package import override %s has been found for file%s `

**Review by lyonlai — CHANGES_REQUESTED** [review]
Sorry for the delay, was very busy last week. So this is more like an alias proto then we could point the import from an existing npm package of other generated code.

This is cool, thanks for your contribution. 

Also a question to you, I prefer to see a test for this feature. Is there any good generated package you know that we could import for a test purpose?

File: registry/registry.go
Comment (higebu) [reply registry/registry.go:347]: Thank you for your review. I will fix this ASAP.

Comment (higebu) [reply registry/registry.go]: @lyonlai I fixed the debug message.

Comment (higebu) [reply registry/registry.go]: I think google-protobuf package is good for testing. I try to write a test. Please wait for a while.

Comment (higebu) [reply registry/registry.go]: @lyonlai I added integration test for ts_package option.

File: integration_tests/package.json
Comment (lyonlai) [inline integration_tests/package.json:31]: wouldn't this also be a `devDependencies`?

File: registry/registry.go
Comment (lyonlai) [reply registry/registry.go]: nice.

File: integration_tests/package.json
Comment (higebu) [reply integration_tests/package.json]: Yes. I fixed this.

**Review by lyonlai — APPROVED** [review]

**Comment by lyonlai** [comment]
Thank you very much for you contribution.

**Comment by lyonlai** [comment]
would you mind to squash the commit into 1 before merge? @higebu

**Comment by higebu** [comment]
@lyonlai Done!

**Comment by lyonlai** [comment]
thank you, will merge once test finished green

**Pull request merged**
